src/Sulu/Bundle/MediaBundle/Tests/Functional/Media/Storage/S3StorageTest.php

```diff
@@ -21,6 +21,11 @@ class S3StorageTest extends SuluTestCase
 {
     protected static $class = S3Kernel::class;
 
+    protected static function getKernelClass(): string
+    {
+        return S3Kernel::class;
+    }
```

Comment:
required because of: https://github.com/symfony/symfony/pull/45414

We still need to define `protected static $class = S3Kernel::class;` for prefer lowest tests.